docs/articles/install/3-install-with-helm.md

```diff
@@ -139,10 +142,29 @@ Testkube requires a variety of secrets to operate. Any required secret that is n
 - testkube-minio-credentials
 - testkube-license
 
+### Testkube License
+
+Enter your license key which can be either an offline or online license. Replace «license» with your license.
+
+For an online license, the format is `XXXXXX-XXXXXX-XXXXXX-XXXXXX-XXXXXX-V3`.
+
+```
+kubectl create secret generic testkube-license --from-literal=LICENSE_KEY=«license»
+```
+
+For an offline license, the format is a long string prefixed with `key/`.
+
+```
+kubectl create secret generic testkube-license --from-literal=license.lic=«license»
```

Comment:
I got this from the excerpted comment from a values.yaml, but it's not very clear for me. Please let me know if I understood it correctly:

```
 # -- Enterprise license file secret reference. Place the license key under the key `LICENSE_KEY` key in the secret, and in case of an offline license place the license file under the key `license.lic` in the same secret. Make sure that the license key file does not have any new line characters at the end of the file.
```